tests/test_to_number.cc

```diff
@@ -420,4 +431,107 @@ TEMPLATE_TEST_CASE("to_number(): Floating-point types, constexpr", "[to_number]"
 }
 */
 
+TEST_CASE("test pow10()", "[to_number]")
```

Comment:
A lot of repetition, table might be better, but readability will probably suffer. So lets keep it as is.